server/server.js

```diff
@@ -1,150 +1,159 @@
-          projectId: request.params.projectId
-        }
-      })
-        .then(updates => response.send('Updated'))
-        .then(() => sequelize.close())
-        .catch(err => console.error(err))
-    );
+  app.put('/api/:projectId/updates/:updateId', (request, response) => {
+    const { updateId } = request.params;
+    const { projectId, postedBy, title, body, likes, pubDates } = request.body;
+    db.modifyUpdates({ updateId }, { projectId, postedBy, title, body, likes, pubDates }, (error, results) => {
+      if (error) {
+        response.status(500).send(error);
+      } else {
+        response.status(202).send('Updated!');
+      }
+    });
   });
 
-  app.delete('/:projectId/updates', (request, response) => {
-    const { Update, sequelizeConnection, sequelize } = initializeSequelize();
-    sequelizeConnection.then(() =>
-      Update.destroy({
-        where: {
-          projectId: request.params.projectId
-        }
-      })
-        .then(updates => response.send('Deleted'))
-        .then(() => sequelize.close())
-        .catch(err => console.error(err))
-    );
+  app.delete('/api/:projectId/updates/:updateId', (request, response) => {
+    const { updateId } = request.params;
+    db.removeUpdates({ updateId }, (error, results) => {
+      if (error) {
+        response.status(500).send(error);
+      } else {
+        response.status(202).send('Deleted!');
+      }
+    });
   });
 }
 
 
-  // app.get('/api/:projectId/updates', (request, response) => {
-  //   db.findUpdates((error, results) => {
-  //     if (error) {
-  //       response.status(500).send(error);
-  //     } else {
-  //       response.status(201).send('Posted!');
-  //     }
-  //   });
+  // const projectId = request.params.id;
+  // let data = {
+  //   title: request.body.title,
+  //   body: request.body.body,
+  //   likes: request.body.likes,
+  //   pubDate: request.body.pubDate
+  // };
+
+
+  // const initializeSequelize = require('../database/db.js');
+
+  // app.get('/:projectId/updates', (request, response) => {
+  //   const { Update, sequelizeConnection, sequelize } = initializeSequelize();
+  //   sequelizeConnection.then(() =>
+  //     Update.findAll({
+  //       where: {
+  //         projectId: request.params.projectId
+  //       }
+  //     })
+  //       .then(updates => response.send(updates))
+  //       .then(() => sequelize.close())
+  //       .catch(err => console.error(err))
+  //   );
   // });
 
-  // app.post('/api/:projectId/updates', (request, response) => {
-  //   let data = {
-  //     title: request.body.title,
-  //     body: request.body.body,
-  //     likes: request.body.likes,
-  //     pubDate: request.body.pubDate
-  //   };
-  //   db.addUpdates(data, (error, results) => {
-  //     if (error) {
-  //       response.status(500).send(error);
-  //     } else {
-  //       response.status(201).send('Posted!');
-  //     }
-  //   });
+  // app.post('/:projectId/updates', (request, response) => {
```

Comment:
Commented code without any description. Please uncomment or remove from the source file!